### ISSUE TITLE
Add tools to Paladin Forensic

### ISSUE THREAD

**Issue by rop12770** [thesourcerer8/hddsuperclone#20]
Hey Guys

Why not add the tool to the Paladin Forensic distribution?
That would create a better distro and potentially increase HDDsuperclone popularity.

**Comment by SomeCrazyGuy** [comment]
I like the idea of making hddsuperclone more popular, but I wonder if we can maintain it. I think the arch user repository (AUR) would be the easiest repo to get hddsuperclone into. If we can spare the time, the debian repository would also open up hddsuperclone to the largest audience. This will take time, currently there is not a stable release of the GPL version of hddsuperclone.

**Comment by thesourcerer8** [comment]
I am supporting to get it shipped to Debian.
It sounds like a good idea to get it into Arch and Paladin, but I have no idea what is necessary for it.

**Comment by SomeCrazyGuy** [comment]
> I am supporting to get it shipped to Debian.
> It sounds like a good idea to get it into Arch and Paladin, but I have no idea what is necessary for it.

I think getting into debian would be the best repo in the long term. In the short term the Ubuntu PPA repos and Arch user repository are easier to get but are not the main repositories for the distribution (thry have to be added or enabled manually) However they offer a way to get the software outside github.

**Comment by rop12770** [comment]
Paladin distro is very well-known and used by many forensic investigators all over the world.
It would be a great way for expanding HDDsuperclone.

**Comment by SomeCrazyGuy** [comment]
> Paladin distro is very well-known and used by many forensic investigators all over the world.
> It would be a great way for expanding HDDsuperclone.

I agree that distro availability will improve hddsuperclone popularity, but there is no stable release of GPL hddsuperclone and no maintainer for the hddsuperclone package on that distribution. If you want to volunteer to build and maintain hddsuperclone on that distribution then I don't think you will here any complaints here.

**Comment by rop12770** [comment]
Unfortunately I don't have the knowledge for that...

**Comment by agathakazar** [comment]
> Paladin distro is very well-known
That is a very interesting statement about a distro that is not even listed on distrowatch. 

I have no idea how you found it and even came to use it (getting through email registration and accepting the spam agreement just to download an ISO is not very nice IMO). 

I've booted their live iso, and they offer nothing except a very clunky GUI to some slightly less ordinarily tools like a specialized forensic version of dd (dc3dd, which is readily available on Debian and other distros as well). But slapping "The world's most popular linux forensic suite" on their wallpaper (another grey pattern) is very cringe, I can't believe mandatory account registration and paying $25 for something you can get with just an ordinary internet connection is very popular. 

As for adding this tool to Paladin distro, I haven't found any sources and/or contribution guidelines, so unless whoever they hire to package their .isos decides to add it not much can be done from the outside. 

As mentioned earlier, it would be nice to get hddsuperclone into debian repos, since a lot of other distros are based off that.
